WordPress/Docs/DB/PreparedSQLStandard.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Prepared SQL"
+    >
+    <standard>
+    <![CDATA[
+    When making direct database queries using $wpdb, you should use $wpdb->prepare(), and use placeholders for all variables used in the query. You should not use variable interpolation or concatenation.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Placeholders with $wpdb->prepare() used for all variables in query.">
+        <![CDATA[
+$val = $_GET['foo'];
+$wpdb->prepare(
+    "SELECT * from table WHERE field = <em>%s</em>",
+    $val
+);
```

Comment:
I don't think the interim variable is needed.
I also think it might help to also highlight the "replacement" to link it more clearly to the placeholder.
Also note the use of single quotes for the query as double quotes are not needed here.

```suggestion
$wpdb->prepare(
    'SELECT * from table
        WHERE field = <em>%s</em>',
    <em>$_GET['foo']</em>
);
```